cni-plugin/integration/run.sh

```diff
@@ -19,14 +23,41 @@ function create_test_lab() {
     k create serviceaccount linkerd-cni
     # TODO(stevej): how can we parameterize this manifest with `version` so we
     # can enable a testing matrix?
-    k create -f manifests/linkerd-cni.yaml
+    # Apply all files in scenario directory. For non-flannel CNIs, this will
+    # include the CNI manifest itself.
+    k apply -f "manifests/$SCENARIO/"
 }
 
+# TODO (matei): skip this based on env var? Useful when running locally to see
```

Comment:
What I do when developing is add a `sleep 60` and then catch it with `Ctrl-Z` to suspend `run.sh` That allows me to inspect all the logs and `describe` everything. Once I've collected enough data, I use `fg` and then cleanup occurs. I'm sure we can think of a cleaner way to do this but probably in a later PR. maybe `kill -TSTP %1`?

Reply:
I'll put a comment in here while I'm making the next test. Nothing to do on your part.